### ISSUE TITLE
Anyone got this working with 4.19 and VS2017?

### ISSUE THREAD

**Issue by islamhaqq** [YetiTech-Studios/UE4GameLiftClientSDK#11]
Hey guys,

Currently having issues running this. Anybody have issues running this on UE4 4.19 and VS2017?

**Comment by ryanjon2040** [comment]
Any error message?

**Comment by kazyuk** [comment]
Everything works! Thanks! I have a question, what about support linux ?

**Comment by SyedAman** [comment]
@kazyuk how about 4.20?

**Comment by kazyuk** [comment]
@SyedAman, yes. Now, we start to use 4.21, but this problem occurred at 4.20. I created issue about it https://github.com/YetiTech-Studios/UE4GameLiftClientSDK/issues/12#issue-363991614. Please, help me :) I needed because we host servers in the Amazon and Linux price is lower than on Windows.
